tools/cli/wskaction.py

```diff
@@ -34,6 +34,7 @@ def __init__(self):
 
     def getItemSpecificCommands(self, parser, props):
         subcmd = parser.add_parser('create', help='create new action')
+        subcmd.add_argument('--type', help='the type of version of the runtime')
```

Comment:
type should accept a parameter which indicated the `type` - the CLI could then validate lightly or defer to the controller.